Firestore/core/test/firebase/firestore/util/CMakeLists.txt

```diff
@@ -14,33 +14,29 @@
 
 cc_test(
   firebase_firestore_util_test
-  autoid_test.cc
-  secure_random_test.cc
```

Comment:
Is secure_random_test.cc is no longer required?

Reply:
This was a bad merge; restored.